packages/odyssey/src/scss/components/_table.scss

```diff
@@ -35,6 +35,7 @@
     padding: $small-spacing $base-spacing;
     text-align: left;
     vertical-align: baseline;
+    overflow-wrap: break-word;
```

Comment:
This seems like the correct solution, but can you describe why breaking is better suited than truncating?

Also - maybe we add some docs to [this page](https://github.com/okta/odyssey/blob/master/packages/docs/source/components/table.md#usage) to explain our reasoning.

Reply:
@jmelberg-okta as a general rule, we avoid truncation whenever possible (for all components)

Reply:
Gotcha. Is this rule defined somewhere? Unless I'm misunderstanding the spirit of the design docs, it seems like a great place to a rule like this.

Reply:
@joejezowski-okta @arnoldsandoval-okta i like the idea of adding a "General Principles" or "Guiding Principles" section in the docs somewhere - things that aren't component specific, but define direction for the DS. worthwhile?